build-container.sh

```diff
@@ -24,11 +24,10 @@ docker buildx build -t webserver-static:${VERSION} -f webserver/Dockerfile --tar
 docker buildx build -t webserver-typo3:${VERSION} -f webserver/Dockerfile --target=webserver-typo3 webserver
 docker buildx build -t webserver-shopware:${VERSION} -f webserver/Dockerfile --target=webserver-shopware webserver
 docker buildx build -t chromium:${VERSION} -f chromium/Dockerfile chromium
-docker buildx build -t mailhog:${VERSION} -f mailhog/Dockerfile mailhog
+docker buildx build -t mailpit:${VERSION} -f mailpit/Dockerfile mailpit
```

Comment:
wolltest du mailhog entfernen?

```suggestion
docker buildx build -t mailhog:${VERSION} -f mailhog/Dockerfile mailhog
docker buildx build -t mailpit:${VERSION} -f mailpit/Dockerfile mailpit
```

Reply:
Yes as mailhog did not update since years.
We can now deprecate it and use mailpit as a drop in replacement.